app/assets/images/homepage/_README.md

```diff
@@ -2,9 +2,9 @@
 
 Images can cause a significant increase in the download time for the homepage. To prevent this:
 
- - resize the image to 608 by 405 pixels. This is the largest that the image will be displayed at.
+ - resize the image to 600 by 400 pixels. This is the largest that the image will be displayed at.
```

Comment:
Nice, good spot

Reply:
Thanks, I ended up a bit confused here whether the documentation was out of date or we'd made a mistake with some of the code. I've reached out to Ian to try confirm it - the number seemed too specific to be a mistake.

Reply:
I don't know where I got 608px from - I suspect it's an out by two error, as the maximum width that an image _could_ be is 610px.

Evidence in case this gets asked again in the future!

<img width="1552" alt="Screenshot 2021-03-04 at 16 20 52" src="https://user-images.githubusercontent.com/1732331/109994872-a4e96600-7d05-11eb-85d1-c576be3f7c06.png">